magento/magento2
# Short-term admin accounts #22833

### Description (*)
Provides the ability to set an expiration date on admin user accounts, after which the account will be disabled.

### Fixed Issues (if relevant)

1. magento/magento2#22833: Short-term admin accounts

### Manual testing scenarios (*)

#### Manual testing scenario 1
1. Create an admin user
2. Set their expiration date to 5 minutes from now
3. Login as that user
5. Wait 5 minutes
4. Navigate to another page
5. User should be logged out

#### Manual testing scenario 2
1. Create an admin user
2. Set their expiration date to 5 minutes from now
3. In 5 minutes, try to login as that user
4. Login should fail with message: "The account sign-in was incorrect or your account is disabled temporarily. Please wait and try again later"

#### Manual testing scenario 3
1. Create an admin user
2. Set their expiration date to 5 minutes from now
3. In 5 minutes, run the Magento cron manually
4. Check the admin user: the account should be disabled and the expiration date should now be empty

### Contribution checklist (*)
